gem_city.rb

```diff
@@ -0,0 +1,55 @@
+# This class represents the town of GemCity
+# This is a town riddled with crime but we can find out how happy the town is
+class GemCity
+  attr_reader :population
+
+  def initialize
+    @people = {
```

Comment:
Splitting these out to `@thieves` and `@officers` will probably simplify things.